javasphinx/apidoc.py

```diff
@@ -192,8 +198,16 @@ def generate_from_source_file(doc_compiler, source_file, cache_dir):
     except Exception:
         util.unexpected('Unexpected exception while parsing %s', source_file)
 
+    documents = {}
     try:
-        documents = doc_compiler.compile(ast)
+        if source_file.endswith("package-info.java"):
```

Comment:
In fact it works. I tried that approach but I was getting an error in Visual Studio Code, complaining that such a "package" attribute didn't exist. But now I understand why: the attributes are created dynamically when the java file is read.